JASP-Engine/JASP/po/R-JASP.pot

```diff
@@ -6,7 +6,7 @@ msgstr ""
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=UTF-8\n"
```

Comment:
Could this have caused that?
@TimKDJ ?

Reply:
No, the .pot file declarations don't do much